sdks/python/README.md

```diff
@@ -257,78 +258,77 @@ Some interesting classes to navigate to:
-p = df.Pipeline('DirectPipelineRunner')
+p = beam.Pipeline('DirectPipelineRunner')
 # Create a PCollection with names and write it to a file.
 (p
- | df.Create('add names', ['Ann', 'Joe'])
- | df.Write('save', df.io.TextFileSink('./names')))
+ | 'add names' >> beam.Create(['Ann', 'Joe'])
+ | 'save' >> beam.io.Write(beam.io.TextFileSink('./names')))
 # Execute the pipeline.
 p.run()
 ```
 
-### Hello world (with Map)
+### Basic pipeline (with Map)
 
 The `Map` transform takes a callable, which will be applied to each
 element of the input `PCollection` and must return an element to go
 into the output `PCollection`.
 
 ```python
-import google.cloud.dataflow as df
-p = df.Pipeline('DirectPipelineRunner')
-# Read file with names, add a greeting for each, and write results.
+import apache_beam as beam
+p = beam.Pipeline('DirectPipelineRunner')
+# Read a file containing names, add a greeting to each name, and write to a file.
 (p
- | df.Read('load messages', df.io.TextFileSource('./names'))
- | df.Map('add greeting',
-          lambda name, msg: '%s %s!' % (msg, name),
-          'Hello')
- | df.Write('save', df.io.TextFileSink('./greetings')))
+ | 'load messages' >> beam.Read(beam.io.TextFileSource('./names'))
+ | 'add greeting' >> beam.Map(lambda name, msg: '%s, %s!' % (msg, name), 'Hello')
+ | 'save' >> beam.Write(beam.io.TextFileSink('./greetings')))
 p.run()
 ```
 
-### Hello world (with FlatMap)
+### Basic pipeline (with FlatMap)
 
 A `FlatMap` is like a `Map` except its callable returns a (possibly
 empty) iterable of elements for the output `PCollection`.
 
 ```python
-import google.cloud.dataflow as df
-p = df.Pipeline('DirectPipelineRunner')
-# Read previous file, add a name to each greeting and write results.
+import apache_beam as beam
+p = beam.Pipeline('DirectPipelineRunner')
+# Read a file containing names, add two greetings to each name, and write to a file.
 (p
- | df.Read('load messages', df.io.TextFileSource('./names'))
- | df.FlatMap('add greetings',
-              lambda name, msgs: ['%s %s!' % (m, name) for m in msgs],
-              ['Hello', 'Hola'])
- | df.Write('save', df.io.TextFileSink('./greetings')))
+ | 'load messages' >> beam.Read(beam.io.TextFileSource('./names'))
+ | 'add greetings' >> beam.FlatMap(
+        lambda name, msgs: ['%s %s!' % (m, name) for m in msgs],
+        ['Hello', 'Hola']
+    )
+ | beam.Write('save', beam.io.TextFileSink('./greetings')))
 p.run()
 ```
 
-### Hello world (with FlatMap and yield)
+### Basic pipeline (with FlatMap and yield)
 
 The callable of a `FlatMap` can be a generator, that is,
 a function using `yield`.
 
 ```python
-import google.cloud.dataflow as df
-p = df.Pipeline('DirectPipelineRunner')
-# Add greetings using a FlatMap function using yield.
+import apache_beam as beam
+p = beam.Pipeline('DirectPipelineRunner')
+# Read a file containing names, add two greetings to each name (using yield), and write to a file.
```

Comment:
I believe having `using a FlatMap function using yield` in the comment was informative. Also please break long lines into two.